src/main/java/com/pizzashop/pizzashop/Controllers/ToppingsController.java

```diff
@@ -2,15 +2,16 @@
 
 import com.pizzashop.pizzashop.Models.Topping;
 import com.pizzashop.pizzashop.Repositories.ToppingsRepository;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.Optional;
 
 @RestController
-@RequestMapping("/toppings")
+@RequestMapping("/api/v1/toppings")
```

Comment:
please use the base path prop

`spring.data.rest.basePath=/api/v1`

`server.contextPath=/api/v1`